cmd/api/src/model/appcfg/parameter.go

```diff
@@ -89,94 +139,69 @@ type ParameterService interface {
 	SetConfigurationParameter(ctx context.Context, configurationParameter Parameter) error
 }
 
-func AvailableParameters() (ParameterSet, error) {
```

Comment:
This was solely used to validate the keys, which has been inlined in the `isValidKey` fn above